### PR TITLE
Chatgpt for developer blog

### PR DESCRIPTION
**Pull request by crystoll**
A new blog article. Ready for review, and of course even approve and merge.

### CODE REVIEW

**Review by filipwastberg — APPROVED** [review]
Think it looks good. I couldn't find any spelling errors. If anything I'd maybe split this into two posts because of length. But this is a great blog post :)

File: _posts/2023-06-09-chatgpt-for-developers.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+![Generating regular expressions](/img/chatgpt-for-developer/generate_regex.png)
+
+By analyzing vast amounts of text data from the web, ChatGPT generates code sequences that mimic the insights of experienced coders. Stuck on a JavaScript function? ChatGPT can help. Unsatisfied with the result? Ask for a rewrite. It's like having a tireless coding partner, always ready to help.
+
+If you're a Python master suddenly thrown into a Ruby project, you can ask ChatGPT questions like "How does array iteration work in Ruby?" or "What's the Ruby equivalent of Python’s list comprehension?" If you're having trouble with your code or tools, ChatGPT is there to offer guidance.
+
+If you are just learning your trade, it can be really powerful to have a knowledgeable "partner" to ask questions from. It can sometimes be daunting to keep on asking "stupid questions" from your colleagues, but ChatGPT is always there to help you out. Some studies already show that people might be more comfortable asking questions from a chatbot than from a human.
```
Comment (tuk-solita) [inline _posts/2023-06-09-chatgpt-for-developers.md:33]: > Some studies already show that people might be more comfortable asking questions from a chatbot than from a human.

Interesting. Maybe you could add a link to a study?

**Review by tuk-solita — COMMENTED** [review]
Great write up 👍🏻

**Review by tuk-solita — APPROVED** [review]

File: _posts/2023-06-09-chatgpt-for-developers.md (hunk 1 above)
Comment (jtac) [reply _posts/2023-06-09-chatgpt-for-developers.md:33]: I think this one is the most famous:
https://jamanetwork.com/journals/jamainternalmedicine/article-abstract/2804309

**Review by jtac — COMMENTED** [review]
I think this is very well written and aligns well with best practices and the solita policy. Great stuff.

**Review by jtac — APPROVED** [review]

**Pull request merged**